styles/pages/Home.module.scss

```diff
@@ -1,24 +1,28 @@
 .home__cta {
-  max-width: 800px;
+  max-width: 1200px;
   margin: 0px auto;
   padding: 15px 20px;
   text-align: center;
 
   h1 {
     font-size: 5rem;
     margin: 15px 0px 0px 0px;
+    color: #edb900;
+    font-family: MedievalSharp, Georgia, serif;
   }
 
   p {
-    color: #8e8e8e;
-    font-size: 22px;
+    color: white;
+    text-align: left;
+    font-size: 32px;
```

Comment:
the 32 looks a bit big on my screen, perhaps max out at 24?

Reply:
haha, i have really bad eyesight. I max out at 24px for smaller screen sizes. Open to toning it back a little.